bundle/manifests/sonataflow-operator.clusterserviceversion.yaml

```diff
@@ -537,4 +537,4 @@ spec:
   minKubeVersion: 1.23.0
   provider:
     name: Red Hat
-  version: latest
+  version: 2.0.0-snapshot
```

Comment:
This is not needed probably.

Reply:
I would refrain from adding these changes to the manifests if they are not related to this feature.

Reply:
As discussed I've opened this [KOGITO-9560](https://issues.redhat.com/browse/KOGITO-9560) to mange this issue in a separate task

Reply:
Are you reverting this change then?